etc/lime-elements.api.md

```diff
@@ -861,6 +861,14 @@ export interface Icon {
 // @public (undocumented)
 export type IconSize = 'x-small' | 'small' | 'medium' | 'large';
 
+// @public
+interface Image_2 {
+    alt: string;
+    loading?: 'lazy' | 'eager';
+    src: string;
+}
+export { Image_2 as Image }

```

Comment:
This file wasn't updated in the first commit, so the commits weren't atomic. I fixed that for you this time 😊